ajax/libs/matterjs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "matterjs",
+  "filename": "matter.min.js",
+  "version": "0.1.3",
+  "description": "A tiny collection of UI components",
+  "homepage": "https://stevenmiller888.github.io/matter",
+  "keywords": [
+    "matter",
+    "matterjs",
+    "components",
+    "deku",
+    "deku-component",
+    "ui"
+  ],
+  "author": {
+    "name": "Steven Miller",
+    "url": "https://github.com/stevenmiller888/"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/stevenmiller888/matter.git"
+  },
+  "npmName": "matterjs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.js"
```

Comment:
We don't know if all the other css files should be excluded, maybe should confirm with author.